domain/src/test/java/org/oppia/android/domain/classify/rules/numberwithunits/NumberWithUnitsIsEquivalentToRuleClassifierProviderTest.kt

```diff
@@ -0,0 +1,173 @@
+package org.oppia.android.domain.classify.rules.numberwithunits
+
+import android.app.Application
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.android.domain.classify.InteractionObjectTestBuilder
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+import kotlin.reflect.KClass
+import kotlin.reflect.full.cast
+import kotlin.test.fail
+
+/** Tests for [NumberWithUnitsIsEquivalentToRuleClassifierProvider]. */
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(manifest = Config.NONE)
+class NumberWithUnitsIsEquivalentToRuleClassifierProviderTest {
+
+  private val FRACTION_1_OVER_2 =
```

Comment:
`FRACTION_1_OVER_2` can be changed to `FRACTION_VALUE_TEST_1_OVER_2 `

Reply:
Done